lib/dxf/parser.rb

```diff
@@ -24,6 +24,12 @@ def parse(io)
         raise ParseError, "Expecting a SECTION, not #{value}" unless 'SECTION' == value
         parse_section(io)
       end
+      self.entities.map.with_index do |entity, i|
+        if (entity.respond_to?(:points))
+          self.entities[i] = entity.lines
+        end
+      end
+      self.entities = self.entities.flatten
```

Comment:
this adjustment changes the parsed result entities from points to Lines and prevents empty instances